tools/build_script_generator/make/resources/Makefile.in

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021, Niklas Hauser
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+APP_SRC := $(wildcard *.cpp)
```

Comment:
Also include `*.c++`?

And maybe even C files?

Reply:
Yes, just need to figure out how to search for nested sources.